crates/orderbook-commons/src/lib.rs

```diff
@@ -21,7 +21,11 @@ pub use crate::price::Price;
 pub use crate::price::Prices;
 
 /// The prefix used in the description field of an order-matching fee invoice to be paid by a taker.
-pub const FEE_INVOICE_DESCRIPTION_PREFIX_TAKER: &str = "taker-fee-";
+pub const FEE_INVOICE_DESCRIPTION_PREFIX_TAKER: &str = "taker-fee";
+
+/// The prefix used in the description field of an JIT channel opening invoice to be paid by the
+/// client.
+pub const JIT_FEE_INVOICE_DESCRIPTION_PREFIX: &str = "jit-channel-fee";
```

Comment:
🤔 Hmmm, this is weird because the JIT channel is orthogonal to the orderbook. Maybe put this in `ln-dlc-node` for now?

Reply:
True, can do!